CMakeLists.txt

```diff
@@ -704,6 +704,8 @@ if(NOT TARGET uninstall)
 		COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
 endif()
 
+option(LICENSE_CHECK "Print out licenses of files used in binaries" OFF)
```

Comment:
The commit message mentions this option but uses the other word variant: LICENCE. Cmake won't recognize option `DLICENCE_CHECK`.